core/ledger/kvledger/txmgmt/statedb/statecouchdb/couchdbutil.go

```diff
@@ -134,19 +146,50 @@ func createCouchDatabase(couchInstance *couchInstance, dbName string) (*couchDat
 	return &couchDBDatabase, nil
 }
 
+// legacy versions of Couch prior to 3.0.0 did not require a userID and password to be configured when starting
+// the database. If the _users table did not exist in Couch < 3.0 the peer startup process created the _users
+// table. In 3.0.0 the _users table cannot be modified directly. This function can be used to determine if legacy (< 3.0.0)
```

Comment:
It might be possible to modify _users if appropriate config is set -- as per the depreciation notice.

I am fine with the current code as it helps to remove having an assumption on the config.

Reply:
I'll clarify the comment, and point to the deprecation notice.